test/ConfigTraitTest.php

```diff
@@ -0,0 +1,154 @@
+<?php
+/**
+ * @see       https://github.com/zendframework/zend-expressive-authentication-oauth2 for the canonical source repository
+ * @copyright Copyright (c) 2017 Zend Technologies USA Inc. (https://www.zend.com)
+ * @license   https://github.com/zendframework/zend-expressive-authentication-oauth2/blob/master/LICENSE.md
+ *     New BSD License
+ */
+
+declare(strict_types=1);
+
+namespace ZendTest\Expressive\Authentication\OAuth2;
+
+use PHPUnit\Framework\TestCase;
+use Psr\Container\ContainerInterface;
+use Zend\Expressive\Authentication\OAuth2\ConfigTrait;
+
+class ConfigTraitTest extends TestCase
+{
+    public function setUp()
+    {
+        $this->trait = $trait = new class {
+            use ConfigTrait;
+
+            public function proxy(string $name, ContainerInterface $container)
+            {
+                return $this->$name($container);
+            }
+        };
+        $this->config = [
+            'authentication' => [
+                'private_key' => 'xxx',
+                'encryption_key' => 'xxx',
+                'access_token_expire' => '3600',
+                'refresh_token_expire' => '3600',
+                'auth_code_expire' => '120',
+                'grants' => ['xxx']
+            ]
+        ];
+        $this->container = $this->prophesize(ContainerInterface::class);
+        $this->container->get('config')
+            ->willReturn($this->config);
+    }
+
+    /**
+     * @expectedException Zend\Expressive\Authentication\OAuth2\Exception\InvalidConfigException
+     */
+    public function testGetPrivateKeyNoConfig()
```

Comment:
Update your test names to indicate what _behavior_ is being tested. In this case, the method would become something like `testGetPrivateKeyWhenNoConfigPresentWillResultInAnException`.

This practice makes understanding the _intention_ behind the test clearer when maintainers are trying to understand a failure later.